integreat_cms/static/src/js/collapsible-boxes.ts

```diff
@@ -26,18 +29,35 @@ const toggleDashboardSection = async ({ target }: Event): Promise<void> => {
         setTimeout(() => {
             content.style.height = "auto";
         }, timeoutDuration);
+        collapsible.removeAttribute("data-collapsed");
     } else {
         // Set height to explicit numerical value instead of "auto" to enable continuous transition
         content.style.height = `${content.clientHeight}px`;
         // Use timeout to allow a smooth transition
         setTimeout(() => {
             content.style.height = "0px";
         });
+        collapsible.dataset.collapsed = "";
     }
+    // Store the currently collapsed boxes in a cookie
+    const collapsedBoxes = Array.from(document.querySelectorAll("[data-collapsed]:not([id=''])")).map((box) => box.id);
+    document.cookie = `collapsed-boxes=${JSON.stringify(collapsedBoxes)}`;
```

Comment:
This is just a preliminary suggestion - it doesn't work since it does not preserve e.g. the state of the page form, when dashboard boxes are toggled and vice versa. But maybe this is a good enough starting point for you to figure something out.

Reply:
Ah, a funny think I just noticed: This nearly works, because the cookie only counts for the current path if not specified otherwise, so the page form cookie does not overwrite the dashboard cookie, but they just are independent of each other and we store one list of ids for each path:

![cookie](https://user-images.githubusercontent.com/16021269/206707321-e425005f-0d9d-43b2-a7ad-efb36191b772.png)

In general, this is a good thing, because it makes things a bit easier - but it also creates one additional problem: The cookie is now valid only for one specific page, but probably we want to collapse the same box on all page forms, independently of the page id.

So probably, we need something along the lines of:
```suggestion
    const collapsedBoxes = Array.from(document.querySelectorAll("[data-collapsed]:not([id=''])")).map((box) => box.id);
    const cookiePath = window.location.pathname.split("/").slice(0, 3).join("/");
    document.cookie = `collapsed-boxes=${JSON.stringify(collapsedBoxes)};path=${cookiePath}`;
```

Reply:
Yes, this solves the issue. I was working on a different idea in the meantime, but my solution only fixed some of the issues. So let's go with your solution. As far as I tested it, it's working reliably now :) Thank you a lot